.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: ":bug: Bug report"
+about: If something isn't working as expected.
```

Comment:
Both `about` sections should be consistent with each other.

The other one is: `I have a suggestion (and may want to implement it)`

How about something like:
```suggestion
about: I got unexpected behavior and think it is a bug
```

Reply:
Makes sense to me 👍🏻